waltid-services/waltid-verifier-api/src/main/kotlin/id/walt/verifier/oidc/VerificationUseCase.kt

```diff
@@ -111,12 +111,13 @@ class VerificationUseCase(
         session
     }
 
-    fun getSession(sessionId: String): PresentationSession = sessionId.let { OIDCVerifierService.getSession(it) }!!
+    fun getSession(sessionId: String): PresentationSession = sessionId.let { OIDCVerifierService.getSession(it) }
+        ?: throw NotFoundException("State parameter doesn't refer to an existing session, or session expired: $sessionId")
```

Comment:
This what the user will get:

`State parameter doesn't refer to an existing session, or session expired: LOREMIPSUM`

Looking at the code, it's clear what the last token of the message is about. However, when the user gets the resulting message, it might not be so clear. What if it was like this?

`State parameter LOREMIPSUM doesn't refer to an existing session or session is expired.`